.kokoro/presubmit/node11/common.cfg

```diff
@@ -0,0 +1,24 @@
+# Format: //devtools/kokoro/config/proto/build.proto
```

Comment:
Pretty sure we dropped testing `node:11` at some point.  We really need to have a single source of truth here.